lib/geordi/commands/drop_databases.rb

```diff
@@ -49,6 +49,6 @@ def drop_databases
   cleaner.clean_postgres unless options.mysql_only
 
   Hint.did_you_know [
-    :delete_dumps
+    :delete_dumps,
```

Comment:
Ok, wouldn't have been necessary to touch _all_ files. Now you have done just what my annotation aimed to prevent: touching lines that have not actually changed :grimacing: But keep this now, it's not too bad.